CefSharp/IBrowser.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CefSharp
+{
+    /// <summary>
+    /// CefSharp interface for CefBrowser.
+    /// </summary>
+    public interface IBrowser
```

Comment:
Any further thoughts on `IBrowser` as a name? It's somewhat of a tricky one, just seems a little too close to `IWebBrowser`